atest/acceptance/keywords/elements.robot

```diff
@@ -55,6 +55,13 @@ Get Element Attribute
     ${class}=    Get Element Attribute    ${second_div}    class
     Should Be Equal    ${class}    Second Class
 
+Get Element Attribute Value Should Be
+    Element Attribute Value Should Be  link=Absolute external link  href  http://www.google.com/
+    Element Attribute Value Should Be  link=Relative to sub-directory  href  http://localhost:7000/html/sub/index.html
+    Element Attribute Value Should Be  id=image_id  src  http://localhost:7000/html/image.jpg
+    ${attribute_value}=  Get Element Attribute  css=#second_div  class
+    Element Attribute Value Should Be  css=#second_div  class  ${attribute_value}
```

Comment:
More test should be written. 
1) Write test where keyword also fails and default error message is used.
2) Write test where custom error message is used.
3) Write test which uses non-ascii characters (like ö, ä or å) in locator. 
4) Write test where the element does not exist.
5) Write test which uses non-ascii characters (like ö, ä or å) in attribute value. 
6) Add test which uses WebElment as locator.